bundler/bundler.go

```diff
@@ -103,14 +127,19 @@ func NewBundler(caBundleFile, intBundleFile string) (*Bundler, error) {
 		}
 	}
 
-	return NewBundlerFromPEM(caBundle, intBundle)
+	return NewBundlerFromPEM(caBundle, intBundle, opt...)
 
 }
 
 // NewBundlerFromPEM creates a new Bundler from PEM-encoded root certificates and
 // intermediate certificates.
 // If caBundlePEM is nil, the resulting Bundler can only do "Force" bundle.
-func NewBundlerFromPEM(caBundlePEM, intBundlePEM []byte) (*Bundler, error) {
+func NewBundlerFromPEM(caBundlePEM, intBundlePEM []byte, opt ...Option) (*Bundler, error) {
```

Comment:
not sure about the semantic of a list of opt functions

Reply:
This is emerging as a common pattern for initializers in Go. Some background: https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis.

And my implementation was inspired (read: mostly copied) from gRPC: https://github.com/grpc/grpc-go/blob/583a6303969ea5075e9bd1dc4b75805dfe66989a/server.go#L328.

Reply:
cool, I learn something new!